pyproject.toml

```diff
@@ -8,11 +8,17 @@ authors = ["xal9wiii4ik <xal9wiii4ik@yandex.ru>"]
 python = "^3.8"
 requests = "^2.26.0"
 Flask = "^2.0.2"
-python-dotenv = "^0.19.2"
 Flask-SQLAlchemy = "^2.5.1"
 psycopg2-binary = "^2.9.2"
 gunicorn = "^20.1.0"
-pre-commit = "^2.16.0"
+Flask-Migrate = "^3.1.0"
+Flask-Script = "^2.0.6"
```

Comment:
you don't need this. flask comes with build-in click library to write scripts

Reply:
done